src/EPiServer.SocialAlloy.Web/Social/Models/ActivityStreams/PageRatingActivity.cs

```diff
@@ -1,24 +1,21 @@
-﻿using EPiServer.SocialAlloy.Web.Social.Adapters;
+﻿using EPiServer.Social.Common;
 
 namespace EPiServer.SocialAlloy.Web.Social.Models
 {
     /// <summary>
-    /// Represents the activity of a user rating being submitted in the site.
+    /// Represents the activity of a page rating being submitted in the site.
     /// </summary>
-    public class PageRatingActivity : CommunityActivity
+    [ExtensionData(TypeId)]
+    public class PageRatingActivity : PageActivity
     {
         /// <summary>
-        /// The rating value that was submitted by a user rating a page.
+        /// The extension data class TypeId.
         /// </summary>
-        public int Value { get; set; }
+        public const string TypeId = "bf41f9e2-50f1-4ee4-bed3-e03f1914f720";
```

Comment:
Do you think we should use strings or Guids for TypeId. I think the flexibility to have either is quite powerful. Or maybe a comment explaining what TypeId is and the possible type of values it could have, since it is new with 2.0?